docs/feature_macros.md

```diff
@@ -2,38 +2,41 @@
 
 Macros allow you to send multiple keystrokes when pressing just one key. QMK has a number of ways to define and use macros. These can do anything you want: type common phrases for you, copypasta, repetitive game movements, or even help you code.
 
-!> **Security Note**: While it is possible to use macros to send passwords, credit card numbers, and other sensitive information it is a supremely bad idea to do so. Anyone who gets a hold of your keyboard will be able to access that information by opening a text editor.
+!> **Security Note:** While it is possible to use macros to send passwords, credit card numbers and other sensitive information, it is a supremely bad idea to do so. Anyone who gets a hold of your keyboard will be able to access that information by opening a text editor.
```

Comment:
Perhaps it should also be noted here that physical access means a determined attacker can potentially read the contents of the flash, so even if you've hidden your password macro behind a maze of layers or combos, it's just security by obscurity.

Reply:
Our local style is to prefer the Oxford comma. It should not be removed.

Reply:
I didn't suggest that?

Reply:
Not sure why I made this change. It's possible that I wanted to rewrite the paragraph, but then forgot to do it. Anyway, I'll revert the comma change. The macros doc isn't the focus of this PR anyway.

I suggest adding such style guidelines (e.g. title casing, serial comma etc.) to `docs/documentation_best_practices.md`, otherwise it's a bit opaque since not all docs use the same style.